src/content/advanced/labelling-workload-clusters/index.md

```diff
@@ -71,6 +71,15 @@ giantswarm.io/monitoring: "false"
 
 You can find more details about this in the [Disable Monitoring]({{< relref "getting-started/observability/monitoring/disable" >}}) article.
 
+### Deletion Prevention
+
+To protect clusters from accidental deletion you can use the following label:
+```nohighlight
+giantswarm.io/prevent-deletion
+```
```

Comment:
Having `metadata` and `labels` there seems a bit verbose and inconsistent with the other sections.
Would it be ok to just add an example value?

Reply:
I thought it makes things clearer since the value alone doesn't prevent people from putting it into an annotation – instead of a label – by mistake. But this is a minor point, not blocking the PR merge.